samples/bot-conversation/python/teamsapp.local.yml

```diff
@@ -0,0 +1,81 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: 1.0.0
+
+provision:
+  # Creates a new Azure Active Directory (AAD) app to authenticate users if the environment variable that stores clientId is empty
+  - uses: aadApp/create 
+    with:
+      name: teamsConversationBot-aad # Note: when you run aadApp/update, the AAD app name will be updated based on the definition in manifest. If you don't want to change the name, make sure the name in AAD manifest is the same with the name defined here.
+      generateClientSecret: true # If the value is false, the action will not generate client secret for you
+      signInAudience: "AzureADandPersonalMicrosoftAccount" # Multitenant
```

Comment:
May use `AzureADMultiTenant` instead, I remember we don't need to enable PersonalMicrosoftAccount access when creating an AAD app for Azure Bot Service

Reply:
Yes, I think `AzureADMultiTenant` is enough.

Reply:
changed